subworkflows/local/generate_downstream_samplesheets/tests/main.nf.test.snap

```diff
@@ -0,0 +1,12 @@
+{
+    "generate_downstream_samplesheets - taxprofiler": {
+        "content": [
+            "./downstream_samplesheets/taxprofiler.csv"
```

Comment:
To reiterate here for the record: as discussed on slack: checksums can't be used can't because the `file()` resolves to the full path, and this will change depending on the environment teh test isn't being executed (local, gitpod, GHA etc), so we are stuck with nr rows or file name (as here) etc.